wee_slack.py

```diff
@@ -2103,6 +2102,10 @@ def handle_rtmstart(login_data, eventrouter):
 
         dbg("connected to {}".format(t.domain))
 
+    else:
+        w.prnt("", "ERROR: Failed connecting to Slack with token {}: {}"
+            .format(metadata.token, login_data["error"]))
+
```

Comment:
Hanging `else` clauses are a bit icky.  I'd much rather have it reverse the check for `login_data["ok"]` and just returning early.  This'll require a reindent, but it's much clearer to read.

Reply:
Sure, that's better.